roles/ceph-mon/tasks/start_monitor.yml

```diff
@@ -5,3 +5,11 @@
     state: started
     enabled: yes
   changed_when: false
+
+# NOTE: This patch only effect Kraken. Fix when reboot machine, the monitor 
+# service do not auto start. Fix proposed upstream: ceph/ceph#14226
```

Comment:
"This fixes a bug when the monitor service does not start automatically after a reboot."